platform/os_windows.go

```diff
@@ -13,13 +13,17 @@ import (
 	"syscall"
 	"time"
 
+	zaplog "github.com/Azure/azure-container-networking/cni/log"
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/platform/windows/adapter"
 	"github.com/Azure/azure-container-networking/platform/windows/adapter/mellanox"
 	"github.com/pkg/errors"
+	"go.uber.org/zap"
 	"golang.org/x/sys/windows"
 )
 
+var logger = zaplog.CNILogger.With(zap.String("component", "platform-windows"))
```

Comment:
how `zaplog.CNILogger` initialized for cns? should be moved as argument to platform client...

Reply:
@paulyufan2  Do we need this if we are passing the zap logger

Reply:
Yes. In the platform only funcs in os_windows.go are zap logged.

var logger = zaplog.CNILogger.With(zap.String("component", "platform-windows"))
logger is used in ClearNetworkConfiguration() only in this file, so we need to initialize this in this file.
I did pass logger as a parameter in ExecutePowerShellCommand()